### ISSUE TITLE
chromium based apps under wayland, mouse freezes

### ISSUE THREAD

**Issue by wassou93** [hyprwm/Hyprland#6667]
### Regression?

Yes

### System Info and Version

<details> <summary>System/Version info</summary>

```sh
Hyprland, built from branch  at commit 9e781040d9067c2711ec2e9f5b47b76ef70762b3  (props: bump version to 0.41.1).
Date: Thu Jun 13 09:54:06 2024
Tag: v0.41.1, commits: 4818

flags: (if any)


System Information:
System name: Linux
Node name: archlinux
Release: 6.9.6-arch1-1
Version: #1 SMP PREEMPT_DYNAMIC Fri, 21 Jun 2024 19:49:19 +0000


GPU information: 
00:02.0 VGA compatible controller [0300]: Intel Corporation Alder Lake-P GT2 [Iris Xe Graphics] [8086:46a6] (rev 0c) (prog-if 00 [VGA controller])
01:00.0 VGA compatible controller [0300]: NVIDIA Corporation GA106M [GeForce RTX 3060 Mobile / Max-Q] [10de:2560] (rev a1) (prog-if 00 [VGA controller])


os-release: NAME="Arch Linux"
PRETTY_NAME="Arch Linux"
ID=arch
BUILD_ID=rolling
ANSI_COLOR="38;2;23;147;209"
HOME_URL="https://archlinux.org/"
DOCUMENTATION_URL="https://wiki.archlinux.org/"
SUPPORT_URL="https://bbs.archlinux.org/"
BUG_REPORT_URL="https://gitlab.archlinux.org/groups/archlinux/-/issues"
PRIVACY_POLICY_URL="https://terms.archlinux.org/docs/privacy-policy/"
LOGO=archlinux-logo


plugins:

======Config-Start======


Config File: /home/wassou/.config/hypr/hyprland.conf: Read Succeeded
#
# ██╗  ██╗██╗   ██╗██████╗ ██████╗
# ██║  ██║╚██╗ ██╔╝██╔══██╗██╔══██╗
# ███████║ ╚████╔╝ ██████╔╝██████╔╝
# ██╔══██║  ╚██╔╝  ██╔═══╝ ██╔══██╗
# ██║  ██║   ██║   ██║     ██║  ██║
# ╚═╝  ╚═╝   ╚═╝   ╚═╝     ╚═╝  ╚═╝
#
# HYPRLAND Config File by Arfan Zubi

# Monitors
#monitor=eDP-1, 1920x1080, 0x1440, 1
#monitor=,highres, 0x0, 1

# Automatic dual setup
monitor=,preferred,auto,auto

# Manual dual setup - 1.5 scaling
#monitor=eDP-2,2560x1600,0x0,1.5
#monitor=HDMI-A-1,1920x1080,1707x0,1

# Execute at launch
#exec-once = hyprpaper

# Execute waypaper to set wallpaper
exec-once = /usr/bin/swww init
exec-once = sleep 1 && /usr/bin/waypaper --restore

exec-once = waybar & kitty
exec-once = udiskie &
exec-once = $HOME/.config/hypr/scripts/sleep.sh
# Needed by obs studio
exec-once = dbus-update-activation-environment --systemd WAYLAND_DISPLAY XDG_CURRENT_DESKTOP

# Source a file (multi-file configs)
# source = ~/.config/hypr/myColors.conf

# Some default env vars.
env = XCURSOR_SIZE,24
env = XCURSOR_THEME,Breeze

env = XDG_CURRENT_DESKTOP,Hyprland
env = XDG_SESSION_TYPE,wayland
env = XDG_SESSION_DESKTOP,Hyprland

env = GDK_BACKEND,wayland,x11
env = SDL_VIDEODRIVER,wayland

# If using NVIDIA graphics card uncomment these 4 lines!
env = LIBVA_DRIVER_NAME,nvidia
env = GBM_BACKEND,nvidia-drm
env = __GLX_VENDOR_LIBRARY_NAME,nvidia
env = WLR_NO_HARDWARE_CURSORS,1

env = QT_QPA_PLATFORM,wayland;xcb
env = QT_QPA_PLATFORMTHEME,qt6ct
env = QT_AUTO_SCREEN_SCALE_FACTOR,1
env = QT_WAYLAND_DISABLE_WINDOWDECORATION,1

input {
    kb_layout = fr
    kb_variant =
    kb_model =
    kb_options =
    kb_rules =

    follow_mouse = 1

    touchpad {
        natural_scroll = no
    }

    sensitivity = 0 # -1.0 - 1.0, 0 means no modification.
}

general {
    gaps_in = 7
    gaps_out = 10
    border_size = 4
    col.active_border = rgba(a7c080ff)
    col.inactive_border = rgba(3d484dff)

    layout = dwindle
}

decoration {
    rounding = 5

    blur {
        enabled = true
        size = 2
        passes = 3
        new_optimizations = true
    }

    drop_shadow = true
    shadow_range = 15
    shadow_render_power = 3
    col.shadow = rgba(1a1a1aee)
}

animations {
    enabled = yes

    bezier = myBezier, 0.05, 0.9, 0.1, 1.05

    animation = windows, 1, 7, myBezier
    animation = windowsOut, 1, 7, default, popin 80%
    animation = border, 1, 10, default
    animation = borderangle, 1, 8, default
    animation = fade, 1, 7, default
    animation = workspaces, 1, 6, default
}

dwindle {
    pseudotile = yes # master switch for pseudotiling. Enabling is bound to mainMod + P in the keybinds section below
    preserve_split = yes # you probably want this
}

master {
    new_is_master = true
}

gestures {
    workspace_swipe = off
}

device {
    name = epic-mouse-v1
    sensitivity = -0.5
}

misc {
    disable_hyprland_logo = true
    disable_splash_rendering = true
    mouse_move_enables_dpms = true
    key_press_enables_dpms = true
}

xwayland {
    force_zero_scaling = true
}

windowrule = opacity 0.9 override,^(kitty)$
windowrule = opacity 0.9 override,^(neovide)$
windowrule = opacity 0.9 override,^(Code)$

# Keybinds
$mainMod = SUPER

bind = $mainMod, Return, exec, kitty
bind = $mainMod, A, exec, alacritty
bind = $mainMod, W, exec, __NV_PRIME_RENDER_OFFLOAD=1 __VK_LAYER_NV_optimus=NVIDIA_only __GLX_VENDOR_LIBRARY_NAME=nvidia firefox
bind = $mainMod, D, exec, wofi
bind = $mainMod, B, exec, pkill -USR1 gammastep || gammastep

bind = $mainMod, F, fullscreen
bind = $mainMod, Q, killactive,
bind = $mainMod SHIFT, E, exit,
bind = $mainMod, V, togglefloating,
bind = $mainMod, P, pseudo, # dwindle
bind = $mainMod, Space, togglesplit, # dwindle


# HYPRSHOT
# Screenshot a window
bind = $mainMod, PRINT, exec, hyprshot -m window
# Screenshot a monitor
bind = , PRINT, exec, hyprshot -m output
# Screenshot a region
bind = $mainMod SHIFT, PRINT, exec, hyprshot -m region

# Move focus with mainMod + [H, L, K, J]
bind = $mainMod, L, movefocus, l
bind = $mainMod, H, movefocus, r
bind = $mainMod, K, movefocus, u
bind = $mainMod, J, movefocus, d

# Move active window with mainMod + SHIFT + [H, L, K, J]
bind = $mainMod SHIFT, H, movewindow, l
bind = $mainMod SHIFT, L, movewindow, r
bind = $mainMod SHIFT, K, movewindow, u
bind = $mainMod SHIFT, J, movewindow, d

# Resize active window with mainMod + ALT + [H, L, K, J]
bind = $mainMod ALT, H, resizeactive, -30 0
bind = $mainMod ALT, L, resizeactive, 30 0
bind = $mainMod ALT, K, resizeactive, 0 -30
bind = $mainMod ALT, J, resizeactive, 0 30

# Switch workspaces with mainMod + [0-9]
bind = $mainMod, 1, workspace, 1
bind = $mainMod, 2, workspace, 2
bind = $mainMod, 3, workspace, 3
bind = $mainMod, 4, workspace, 4
bind = $mainMod, 5, workspace, 5
bind = $mainMod, 6, workspace, 6
bind = $mainMod, 7, workspace, 7
bind = $mainMod, 8, workspace, 8
bind = $mainMod, 9, workspace, 9
bind = $mainMod, 0, workspace, 10

# Move active window to a workspace with mainMod + SHIFT + [0-9]
bind = $mainMod SHIFT, 1, movetoworkspace, 1
bind = $mainMod SHIFT, 2, movetoworkspace, 2
bind = $mainMod SHIFT, 3, movetoworkspace, 3
bind = $mainMod SHIFT, 4, movetoworkspace, 4
bind = $mainMod SHIFT, 5, movetoworkspace, 5
bind = $mainMod SHIFT, 6, movetoworkspace, 6
bind = $mainMod SHIFT, 7, movetoworkspace, 7
bind = $mainMod SHIFT, 8, movetoworkspace, 8
bind = $mainMod SHIFT, 9, movetoworkspace, 9
bind = $mainMod SHIFT, 0, movetoworkspace, 10

# Scroll through existing workspaces with mainMod + scroll
bind = $mainMod, mouse_down, workspace, e+1
bind = $mainMod, mouse_up, workspace, e-1

# Move/resize windows with mainMod + LMB/RMB and dragging
bindm = $mainMod, mouse:272, movewindow
bindm = $mainMod, mouse:273, resizewindow




======Config-End========
```

</details>

### Description

Whenever I'm using chrome under wayland (setting Preferred Ozone platform to 'auto').

I have multiple tabs open in one of the tabs I select some text then I try to move it outside chrome it works. but when I drop it inside the same tab the mouse gets frozen and I can't use it anymore.

This is very similar to this KDE bug: https://bugs.kde.org/show_bug.cgi?id=482142

This was resolved in kde 6.1 and back ported to older versions last I tried in kde I confirmed it's resolved.

### How to reproduce

Open a tab in chromium and select a text then move it with the mouse and drop it in the same tab window.

The mouse will become stuck at the arrow and clicks won't have any effects inside that tab, whenever you change to the next tab you can control it fine.

### Crash reports, logs, images, videos

Uploaded a video on youtube to showcase the issue:
https://youtu.be/23dEYJEv7ow

**Comment by Liassica** [comment]
Can reproduce on Chromium, Discord, and FreeTube on Hyprland 0.41.1.

**Comment by Liassica** [comment]
Seems to be fixed in -git, https://github.com/hyprwm/Hyprland/issues/6660#issuecomment-2188768067

**Comment by wassou93** [comment]
> Seems to be fixed in -git, [#6660 (comment)](https://github.com/hyprwm/Hyprland/issues/6660#issuecomment-2188768067)

Thanks for letting me know I'll try the git version then.

**Comment by wassou93** [comment]
I can confirm issue is resolved in the git version. closing issue
